CONTRIBUTING.rst

```diff
@@ -0,0 +1,65 @@
+Contributions
+=============
+
+Contributions (including bug reports, fixes, and features additions) are welcome
+from the public open source community subject to the terms in the
+`Brown University Software License`_. For any contribution, please first
+create an issue describing current behavior and if applicable, the change
+proposed. Changes should be submitted in the form of pull requests
+that reference the previously created issue.
+
+Once the implementation of a piece of functionality is considered to be bug
+free it can be incorporated into the master branch.
+
+All users and contributors to HNN are expected to follow our `code of conduct`_
+
+To help developing HNN, you will need a few adjustments to your
+installation as shown below.
+
+Running tests
+-------------
+
+To run the tests using ``pytest``, you need to have the git cloned ``hnn``
+repository with an editable pip install::
+
+    $ git clone https://github.com/jonescompneurolab/hnn --depth 1
+    $ cd hnn
+    $ python setup.py develop
+
+Then, install the following python packages::
+
+    $ pip install flake8 pytest pytest-cov
+
+The following tests should pass for contributions to be merged::
+
+    $ flake8 --count --exclude  __init__.py
+    $ py.test hnn/tests/
+
+Creating releases
+-----------------
+
+After every release creation, assets (``.tar.gz`` and ``.zip`` files) need to be uploaded
+to GitHub because those are the files that users will download when following our installation
+instructions. This will ideally be replaced by a ``pip``-based installation in the future. For
+now, using assets for each release allow download counts to be retrieved using the GitHub API.
+Note, that the automatically created assets with each release, named "Source code" do not have
+download stats tracked. Also the installation instructions point directly to the uploaded
+assets named ``hnn.tar.gz`` and ``hnn.zip``.
+
+**Steps to create a new release**:
```

Comment:
do the installation instructions need to be updated when creating a release?

Reply:
Nope